pkg/registry/service/registry.go

```diff
@@ -36,8 +33,8 @@ type Registry interface {
 	WatchServices(ctx api.Context, labels labels.Selector, fields fields.Selector, resourceVersion string) (watch.Interface, error)
 }
 
-// IPRegistry is a registry that can retrieve or persist a RangeAllocation object.
-type IPRegistry interface {
+// RangeRegistry is a registry that can retrieve or persist a RangeAllocation object.
+type RangeRegistry interface {
```

Comment:
Can you add a TODO here to move this to a general location (other components may need allocation in the future, and it's not service specific).  Doesn't need to be moved here though.